### PR TITLE
account link file

### PR DESCRIPTION
**Pull request by sc454**

### CODE REVIEW

File: accountlink.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+get user's access token
+authenticate access token on ticketmaster
+log into user's account and find card details
```
Comment (jasonchitlaTM) [inline accountlink.txt]: doesnt work! :(((((( 👎

File: accountlink.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+ask for confirmation
```
Comment (jasonchitlaTM) [inline accountlink.txt]: fix confirmation (just capitalize it)

File: accountlink.txt (hunk 1 above)
Comment (sc454) [inline accountlink.txt]: changed it

**Comment by sc454** [comment]
@jasonchitlaTM made changes to be reviewed

**Pull request merged**